tenets/codelingo/k8/use-informer-not-list/codelingo.yaml

```diff
@@ -0,0 +1,18 @@
+tenets:
```

Comment:
This tenet just finds funcs with bool params ?

Reply:
Sorry, this tenet was not supposed to make it into the PR